tests/gnrc_tcp/tests/04-receive_data.py

```diff
@@ -42,10 +45,11 @@ def testfunc(child):
     child.expect_exact('gnrc_tcp_open_active: returns 0')
 
     # Accept Data sent by the host system
-    child.sendline('gnrc_tcp_recv 1000000 ' + str(data_len))
+    child.sendline('gnrc_tcp_recv 20000000 ' + str(data_len))
```

Comment:
Why this change? It should remain lower then the `expect` timeout. Additionally, the test still failed when I set the `expect` timeout to e.g. 10, so increasing this timeout could make this test less reliable.

Reply:
If this timeout fires the test is failed anyway. It is just there to wait for data instead of an immediately return with -EAGAIN. I though aligning it with the pexpect timeout is okay because this timeout should not fire anyway.